Codeinwp/neve
# Elementor auto set full width page templates

### Summary
Automatically set full width page templates in elementor.


### Will affect visual aspect of the product

NO

### Test instructions


- Create a new page and edit with elementor.
- Full Width should be automatically set.


Closes #98.
